bootstrap/kubeadm/internal/cloudinit/controlplane_init.go

```diff
@@ -52,6 +54,11 @@ type ControlPlaneInput struct {
 // NewInitControlPlane returns the user data string to be used on a controlplane instance.
 func NewInitControlPlane(input *ControlPlaneInput) ([]byte, error) {
 	input.Header = cloudConfigHeader
+	if input.KubeadmVerbosity == "0" {
+		input.KubeadmVerbosity = ""
+	} else {
+		input.KubeadmVerbosity = fmt.Sprintf("--v %s", input.KubeadmVerbosity)
+	}
```

Comment:
```golang
	input.KubeadmVerbosity = fmt.Sprintf("--v %s", input.KubeadmVerbosity)
	if input.KubeadmVerbosity == "0" {
		input.KubeadmVerbosity = ""
	}
```

Reply:
this wouldn't work since input.KubeadmVerbosity will have been modified by the time we execute line 2. We could do 
```
	input.KubeadmVerbosity = fmt.Sprintf("--v %s", input.KubeadmVerbosity)
	if input.KubeadmVerbosity == "--v 0" {
		input.KubeadmVerbosity = ""
	}
```
But I find it more readable to do the explicit if/else